rocAL/rocAL_pybind/rocal_pybind.cpp

```diff
@@ -153,6 +154,31 @@ namespace rocal
             output_tensor.copy_data(ptr);
             }
             )
+
+            .def("copy_data_numpy", [](rocalTensor &output_tensor, py::array_t<unsigned char> array) {
```

Comment:
Have we checked if we can make these 2 Lamda functions generic i.e if this can be integrated into a single function @Akilesh2 ?

Reply:
Have we tried to use the keyword "auto" instead if py::array_t<unsigned char> array / py::array_t<float> array.

Try this & let me know